.circleci/config.yml

```diff
@@ -313,8 +313,6 @@ jobs:
               export EMSCRIPTEN=$HOME/emsdk/fastcomp/emscripten
               nvm use v11.9.0
               ./build_js.sh
```

Comment:
```suggestion
              ./build_js.sh --bullet
```

Reply:
Should we add steps for both `./build_js.sh` and `./build_js.sh --bullet` in circleci?

Reply:
Absolutely @Ram81 as long we have tests that will run on both of them.